src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
The terminology doesn't seem quite right here. What actually being tested is a validity of a [_media feature **name**_](https://www.w3.org/TR/mediaqueries-4/#mq-features), not the whole *media feature*.

So, I think we either rename this to `isStandardSyntaxMediaFeatureName` or we adapt this util to check the validity of the whole media feature e.g. `(min-width: 10px)`. I prefer the latter as this rule targets *media features* and so we should exclude anything that isn't a media feature *with standard syntax*.

Is the terminology right in the parser itself?